src/commands/cmd_bit.cc

```diff
@@ -225,10 +227,145 @@ class CommandBitOp : public Commander {
   BitOpFlags op_flag_;
 };
 
+class CommandBitfield : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    BitfieldOperation cmd;
+
+    read_only_ = true;
+    // BITFIELD <key> [commands...]
+    for (auto group = args.begin() + 2; group != args.end();) {
```

Comment:
Though this code LGTM, we need to check the argument of the number of `group` doesn't visit the `args.end()`. Would you mind just using the `Parser` here?

Reply:
Oh seems `size` is already checked, you can decide whether using parser or current code.